src/styles/main.scss

```diff
@@ -0,0 +1,145 @@
+.card-container {
+  width: 100%;
+  box-sizing: border-box;
+  color: #060b35;
+  gap: 46px 48px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid using shorthand properties that set values for other properties. Here, 'gap' is a shorthand for 'row-gap' and 'column-gap'. Consider using the longhand properties instead.